solr/solr-ref-guide/src/the-terms-component.adoc

```diff
@@ -349,11 +349,11 @@ The TermsComponent also supports distributed indexes. For the `/terms` request h
 `shards`::
 Specifies the shards in your distributed indexing configuration. For more information about distributed indexing, see <<distributed-search-with-index-sharding.adoc#,Distributed Search with Index Sharding>>.
 +
-The `shards` parameter is subject to a host whitelist that has to be configured in the component's parameters using the configuration key `shardsWhitelist` and the list of hosts as values.
+The `shards` parameter is subject to a host allow-list that has to be configured in the component's parameters using the configuration key `allowUrls` and the list of hosts as values.
```

Comment:
Information on this page is obsolete; we can remove `shards` here.  See https://issues.apache.org/jira/browse/SOLR-14036

Reply:
I'll remove both `shards` and `shards.qt`.